Reference-Platform/CECommon/OE.md

```diff
@@ -1,6 +1,7 @@
 ## OpenEmbedded and Yocto
 
-This page provides instructions to get started with OpenEmbedded and the Yocto Project on the DragonBoard 410c and HiKey. 
+This page provides instructions to get started with OpenEmbedded and the Yocto Project. It aims at being generic for any board support.
+The board diversity should be addressed through dedicated BSP layer then MACHINE choice.
```

Comment:
I agree that this page needs some improvements. It started as a copy/paste of my initial OE setup page for QCOM boards. However I don't think we want that page to be a reference page for 'any board'. This is the Linaro OE RPB getting started page, so the focus is on the BSP that *we* support, not any. So I think the intro should reflect that, and maybe explain what the OE RPB is.